### ISSUE TITLE
problème dans les datas choisis 

### ISSUE THREAD

**Issue by clothilde-ferreira** [brainhack-ch/piriform-cortex-diversity#24]
bonjour, 

j'ai exporter la table des donner prise par l'algorithme et les données dedans ne corresponde pas au donner que j'ai sur mes excel. 
est ce normal ? 
Exemple pour le features :"Filament BoundingBoxAA Length X" mon excel de raw data dit "346,12" et dans la table de donner pris dans l'algorithme c'est "-0.430301549624355"
je vous met une capture d'écrans

merci à vous 
<img width="1379" alt="Capture d’écran 2023-02-27 à 12 01 56" src="https://user-images.githubusercontent.com/119606166/221546958-64d136d8-39f3-4266-ad21-507a4fc33bc8.png">

**Comment by hugofluhr** [comment]
il me semble que c'est parce que les données sont normalisées/z-scored dans le code avant le clustering.
@Cionkito tu peux confirmer?

**Comment by acionca** [comment]
Yes, c'est exactement ça - est-ce que (@clothilde-ferreira) tu voulais exporter les données avant clustering pour une raison particulière ?

**Comment by clothilde-ferreira** [comment]
c'etait pour comprendre les features qui manquaient suivant les neurones car j'avais le message d'erreur et je voulais avoir le tableau pour voir en globalité quest ce qui manquer pour chaque neurone .

**Comment by acionca** [comment]
Hmm, alors ça pourraît être une idée d'ajouter une option pour sauvegarder les data utilisées dans le clustering en format RAW et non normalisées - en ce qui concerne les messages d'erreurs, on peut continuer la discussion sur l'autre issue correspondante (#23).
